src/ci/scripts/free-disk-space.sh

```diff
@@ -0,0 +1,218 @@
+#!/bin/bash
+# Free disk space on Linux GitHub action runners
```

Comment:
Let's write at the beginning of the file something like `script is inspired by https://github.com/jlumbroso/free-disk-space`, to give the author the credit they deserve 👍